src/sql/base/browser/ui/table/media/slick.grid.css

```diff
@@ -198,3 +198,7 @@
 	display: inline-block;
 	flex: 1 1 auto;
 }
+
+.slick-cell.selected .codicon.toggle {
```

Comment:
I'd recommend to add this color to the theme color instead of hard coding it.
at a minimum, you should add hc-black, and meanwhile make sure this color meets the contrast ratio requirement for all 3 themes.